.github/ISSUE_TEMPLATE/license-exception.md

```diff
@@ -7,9 +7,12 @@ assignees: caniszczyk, amye
 ---
 
 For all license exception requests, we need the following information in order to help make a decision to grant an exception.
-- [ ] What is the alternative license?
-- [ ] If this is newly created code, why can't it be under Apache-2.0? 
-- [ ] Is this an existing 3rd party open source project?
-- [ ] How does this code integrate with or interact with, if at all, other components of CNCF? 
-- [ ] How will the code be maintained? Who is responsible? 
-- [ ] How will the code be kept up to date with security patches?
+- [ ] What is the third party component (name, URL, and version if available)?
+- [ ] What license(s) is it provided under?
+- [ ] If it is Apache-2.0 for code or CC-BY-4.0 for documentation, no further exception request is required.
+- [ ] Does it meet the Allowlist policy? If so, no further exception request is required.
```

Comment:
let's add a link to the policy? https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md